### ISSUE TITLE
Amazon Games library authentication does not redirect to login

### ISSUE THREAD

**Issue by Foshitz** [JosefNemec/PlayniteExtensions#184]
**Check for existing issue**
No current issues open with "Amazon" in title.

**Describe the bug**
When clicking on "Authenticate", a pop-up window comes up but no login comes. Nothing at all shows, only the theme background color.
The "Authenticate"-button for Amazon Games links to the URL:
_`"https://www.amazon.com/ap/signin?openid.ns=http://specs.openid.net/auth/2.0&openid.claimed_id=http://specs.openid.net/auth/2.0/identifier_select&openid.identity=http://specs.openid.net/auth/2.0/identifier_select&openid.mode=checkid_setup&openid.oa2.scope=device_auth_access&openid.ns.oa2=http://www.amazon.com/ap/ext/oauth/2&openid.oa2.response_type=token&openid.oa2.client_id=device:6330386435633439383366623032393938313066303133343139383335313266234132554D56484F58375550345637&language=en_US&marketPlaceId=ATVPDKIKX0DER&openid.return_to=https://www.amazon.com&openid.pape.max_auth_age=0&openid.assoc_handle=amzn_sonic_games_launcher&pageId=amzn_sonic_games_launcher"`_

**To Reproduce**
Issue is present both when trying to login again, after losing authentication, and also first time login after reinstalling the library extension (add-on).

**Screenshots**
![authentication-popup](https://user-images.githubusercontent.com/16353951/170633790-7768b220-485e-43d6-b5af-06881430de9d.jpg)

**Diagnostics ID**
de21ea03-ba82-40ba-89ea-d287aef0cc92

**Comment by Foshitz** [comment]
It works without any issues on my laptop, on the same network.
Any idea on what I can do to fix it on my desktop?

**Comment by JosefNemec** [comment]
Update GPU drivers:
https://github.com/JosefNemec/Playnite/wiki/Known-Issues#integration-logins-dont-work-show-only-whiteblack-screen

**Comment by JosefNemec** [comment]
Try reinstalling Playnite and check that antivirus is not blocking any Playnite processes, including CefSharp.BrowserSubprocess.exe

**Comment by Foshitz** [comment]
I did a clean reinstall of Playnite, no change.
Also checked firewall and antivirus, nothing blocked by antivirus and CefSharp.BrowserSubprocess.exe is allowed in firewall.

Now, after solving the account issues I got because of this Playnite issue, I can login but Playnite doesn´t recognize or remember that I have logged in and authenticated.
Playnite claims it "requires authentication" even though I have logged in several times, and even restarted Playnite between the last 5 tries.
